test/python/transpiler/test_elide_permutations.py

```diff
@@ -235,8 +235,7 @@ def test_partial_permutation_in_middle(self):
 
         # Make sure that the transpiled circuit *with* the final permutation
         # is equivalent to the original circuit
-        perm = pass_.property_set["virtual_permutation_layout"].to_permutation(qc.qubits)
-        res.append(PermutationGate(perm), [0, 1, 2, 3, 4])
+        res.append(PermutationGate(res.layout.routing_permutation()), [0, 1, 2, 3, 4])
```

Comment:
`PassManager.run` explicitly clears out `virtual_permutation_layout` from a `property_set` after execution, since it updates the layout fields.  This new version of the `PermutationGate` is more properly the behaviour we _want_ to test, though.